astropy/units/quantity_helper/scipy_special.py

```diff
@@ -16,11 +16,13 @@
 
 # ufuncs that require dimensionless input and give dimensionless output.
 dimensionless_to_dimensionless_sps_ufuncs = (
-    'erf', 'gamma', 'gammasgn', 'psi', 'rgamma', 'erfc', 'erfcx', 'erfi',
+    'erf','erfc', 'erfcx', 'erfi', 'erfinv', 'erfcinv', 'gamma', 'gammaln',
+    'loggamma', 'gammasgn', 'psi', 'rgamma', 'digamma',
```

Comment:
I'm not quite sure about `gammaln` and `loggamma` - arguably, since they return the natural logarithm, perhaps they should return a functional log-based unit? (Would be a new one, since we have only `dex` for log-10 based...). But then, again `np.log(dimensionless)` gives back dimensionless, so probably best to let them go in...

`rgamma` is fine, as are `digamma`